packages/core/src/routes/experience/classes/experience-interaction.ts

```diff
@@ -423,7 +423,7 @@ export default class ExperienceInteraction {
 
     // MFA fulfilled
     if (!this.hasVerifiedSsoIdentity) {
-      await this.mfa.assertUserMandatoryMfaFulfilled();
+      await this.mfa.assertUserMandatoryMfaFulfilled(this.#interactionEvent);
```

Comment:
The mfa instance has the `interactionContext` access internally. No need to pass in the `interactionEvent` here.